fud/fud/stages/xilinx/xclbin.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, config):
         self.remote_exec = RemoteExecution(self)
         self.temp_location = self.config["stages", self.name, "temp_location"]
 
+        self.save_temps = bool(self.config["stages", self.name, "save_temps"])
```

Comment:
Maybe a comment explaining why this is added would prove useful for future readers. It wasn't obvious to me until I read your PR description.

Reply:
Good call; added a short comment.